roles/reconcile/tasks/reconcile_pods.yaml

```diff
@@ -23,27 +48,40 @@
   set_fact:
     etcd_cluster_members: "{{ members }}"
   when: (etcd_cluster_phase == "existing" and etcd_secure_client == "present")
+  ignore_errors: yes
```

Comment:
just curious, were these `ignore_errors` the only solution? or may be the best way to do it?

Reply:
Yeah couldn't any other way!